AirGradient-DIY/AirGradient-DIY.ino

```diff
@@ -18,6 +18,9 @@ AirGradient ag = AirGradient();
 // Optional.
 const char* deviceId = "";
 
+// set to true to switch display from Celcius to Fahrenheit
+boolean inF = false;
```

Comment:
I think to make this simpler, could you change the var to a string, like `temp_display`, with default of `"C"`, and switch the logic below to use that?

Reply:
It is done